go_modules/lib/dependabot/go_modules/update_checker.rb

```diff
@@ -112,6 +112,8 @@ def updated_dependencies_after_full_unlock
       # Override the base class's check for whether this is a git dependency,
       # since not all dep git dependencies have a SHA version (sometimes their
       # version is the tag)
+      #
+      # TODO should this be removed entirely? I'm not sure what it referred to originally, and this functionality doesn't ring a bell for `go mod`...
```

Comment:
I never used `go dep`, so unclear what the original thing this was trying to guard against... as a result, I'm unclear if this is still applicable to `go mod`... @jurre any idea?

Reply:
In `go mod` versions can also be a tag AFAIK so we'll want to keep this code in

Reply:
ah, I see what it's getting at... updated the comment